README.md

```diff
@@ -154,7 +188,7 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+lena = Person.create(name: "Lena", age: 23)
 ```
 
 Write Ruby code that will query for any person that is 15 years of age.
```

Comment:
Don't see an answer for this one. What Active Record method(s) would you use here?